api/nomics.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import calendar
+import connexion
+
+from services.bitshares_websocket_client import client as bitshares_ws_client
+from services.bitshares_elasticsearch_client import client as bitshares_es_client
+from services.cache import cache
+import api.explorer
+import es_wrapper
+import config
+
+
+def info():
+    return {
+        "name": "Bitshares",
+        "description": "An exchange description of at least 1000 characters in plain text (no html)",
+        "location": "Anywhere",
+        "logo": "https://example.com/exchange-logo.png",
+        "website": "https://bitshares.org/",
+        "twitter": "example",
```

Comment:
bitshares

Isn't [this one](https://twitter.com/bitshares) the official?

Reply:
added at https://github.com/bitshares/bitshares-explorer-api/pull/64/commits/5edc75a9e3d55d6c0e86dc2029d2e41cc2bae48b